include/functions.h

```diff
@@ -660,120 +660,139 @@ s32 func_8005AA90(Camera*, s32, Vec3f*);
+s32 Collider_TrisItemSetAC(GlobalContext* globalCtx, ColliderTrisItem* item);
+s32 Collider_TrisItemSetOC(GlobalContext* globalCtx, ColliderTrisItem* item);
+s32 Collider_InitTris(GlobalContext* globalCtx, ColliderTris* tris);
+s32 Collider_FreeTris(GlobalContext* globalCtx, ColliderTris* tris);
+s32 Collider_DestroyTris(GlobalContext* globalCtx, ColliderTris* tris);
+s32 Collider_SetTris_Set3(GlobalContext* globalCtx, ColliderTris* dest, Actor* actor, ColliderTrisInit_set3* src);
+s32 Collider_SetTris_Set5(GlobalContext* globalCtx, ColliderTris* dest, Actor* actor, ColliderTrisInit* src);
+s32 Collider_SetTris(GlobalContext* globalCtx, ColliderTris* dest, Actor* actor, ColliderTrisInit* src,
+    ColliderTrisItem* list);
+s32 Collider_TrisSetAT(GlobalContext* globalCtx, Collider* collider);
+s32 Collider_TrisSetAC(GlobalContext* globalCtx, Collider* collider);
+s32 Collider_TrisSetOC(GlobalContext* globalCtx, Collider* collider);
+s32 Collider_InitColliderQuadDim(GlobalContext* globalCtx, ColliderQuadDim* dim);
+s32 Collider_DestroyColliderQuadDim(GlobalContext* globalCtx, ColliderQuadDim* dim);
+s32 func_8005CEC4(GlobalContext* globalCtx, ColliderQuadDim* dim);
+void func_8005CEDC(ColliderQuadDim* dim);
+s32 Collider_SetColliderQuadDim(GlobalContext* globalCtx, ColliderQuadDim* dest, ColliderQuadDimInit* src);
+s32 Collider_InitQuad(GlobalContext* globalCtx, ColliderQuad* collision);
+s32 Collider_DestroyQuad(GlobalContext* globalCtx, ColliderQuad* collision);
+s32 Collider_SetQuad_Set3(GlobalContext* globalCtx, ColliderQuad* collision, Actor* actor, ColliderQuadInit_set3* src);
+s32 Collider_SetQuad(GlobalContext* globalCtx, ColliderQuad* collision, Actor* actor, ColliderQuadInit* src);
+s32 Collider_QuadSetAT(GlobalContext* globalCtx, Collider* collider);
+s32 Collider_QuadSetAC(GlobalContext* globalCtx, Collider* collider);
+s32 Collider_QuadSetOC(GlobalContext* globalCtx, Collider* collider);
+s32 func_8005D218(GlobalContext* globalCtx, ColliderQuad* quad, Vec3f* arg2);
+s32 Collider_InitOcLine(GlobalContext* globalCtx, OcLine_s* line);
+s32 Collider_DestroyOcLine(GlobalContext* globalCtx, OcLine_s* line);
 // ? func_8005D334(?);
 // ? func_8005D3A4(?);
-// ? func_8005D3BC(?);
-void func_8005D400(GlobalContext*, SubGlobalContext11E60*);
-// ? func_8005D40C(?);
-// ? func_8005D4DC(?);
-// ? func_8005D62C(?);
-s32 Actor_CollisionCheck_SetAT(GlobalContext* globalCtx, SubGlobalContext11E60* simpleBodyGroups,
-                               ColliderCylinderMain* collision);
-s32 Actor_CollisionCheck_SetAC(GlobalContext* globalCtx, SubGlobalContext11E60* simpleBodyGroups,
-                               ColliderCylinderMain* collision);
-s32 Actor_CollisionCheck_SetOT(GlobalContext* globalCtx, SubGlobalContext11E60* simpleBodyGroups,
-                               ColliderCylinderMain* collision);
+void func_8005D3BC(GlobalContext* globalCtx, CollisionCheckContext* check);
```

Comment:
Imo `check` is misleading as the name for these function arguments, I suggest something clearer and consistent with the rest of the codebase, like `clCheckCtx`.

Might also want to consider naming the global context member the same as these arguments to avoid any confusion. That's how it's done for essentially all other contexts at least.